packages/components/pie-cookie-banner/src/localisation-utils.ts

```diff
@@ -1,11 +1,6 @@
-import {
-    TemplateResult,
-} from 'lit';
-
-import {
-    type CookieBannerLocale,
-    type CustomTagEnhancers,
-} from './defs';
+import { type TemplateResult } from 'lit';
```

Comment:
Looks good to me! Just to clarify, you removed return types and `type TemplateResult` in this file:`packages/components/pie-cookie-banner/src/index.ts`, but didn't remove here, for `enhanceCustomTags` and `localiseRichText` functions. What's the reason for this?

Reply:
Good question! I've taken a look and I guess the short answer is that the return type wasn't providing any value in those other files, because the functions were very simple and the return type is easily inferred because they just return HTML strings.

In this file, the functions are a bit more complex and the types are also used inside the JSDoc comments. They could probably be removed but I don't have a deep enough understanding of the functions at the moment to improve the documentation. If I just removed them we would be losing some potentially helpful comments.